internal/services/oracledatabase/cloud_vm_cluster_helpers.go

```diff
@@ -0,0 +1,71 @@
+package oracledatabase
+
+import (
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2024-06-01/cloudvmclusters"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"strings"
+)
+
+func ConvertDataCollectionOptionsToInternal(dataCollectionOptions *cloudvmclusters.DataCollectionOptions) []DataCollectionOptionsModel {
```

Comment:
These seem to be the expand/flatten functions? these should be in the same file they are used in & use the terminology the rest of the provider does

flatten functions take api response/data -> terraform/internal
expand functions take internal/tf -> api request

Reply:
Thanks, done.